experimental/packages/opentelemetry-instrumentation-http/src/http.ts

```diff
@@ -117,39 +117,49 @@ export class HttpInstrumentation extends InstrumentationBase<Http> {
       ['*'],
       moduleExports => {
         this._diag.debug(`Applying patch for http@${version}`);
-        if (isWrapped(moduleExports.request)) {
-          this._unwrap(moduleExports, 'request');
-        }
-        this._wrap(
-          moduleExports,
-          'request',
-          this._getPatchOutgoingRequestFunction('http')
-        );
-        if (isWrapped(moduleExports.get)) {
-          this._unwrap(moduleExports, 'get');
+        if (this._getConfig().instrumentOutgoingRequests !== false) {
```

Comment:
Since your PR, there has been a change from `_getConfig()` to `getConfig()` that you'll need to update for. Let me know if you need help with that.

Reply:
all good I believe, I rebased on master and incorporated the changes, should be OK!